Makefile

```diff
@@ -45,7 +45,10 @@ endif
 MAKEFLAGS += --no-print-directory
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
-HOSTGO := env -u GOOS -u GOARCH -u GOARM -- go
+HOSTGO := env -u $(GOOS) -u $(GOARCH) -u $(GOARM) -- go
+ifeq ($(GOOS), windows)
+	EXEEXT := .exe
+endif
```

Comment:
At the very top of the makefile we already have a check for if windows_nt can this go up there?

Reply:
We could move it, would also have to move `GOOS ?= $(shell go env GOOS)` as well then. The `windows_nt` also checks against `$OS` opposed to `$GOOS` so it would still be a separate check, so not sure if it would make things cleaner?

Reply:
Oh maybe I misunderstood, I thought `Windows_NT` was specifically for that version of windows but I just read `Windows_NT` matches windows 10. I'll move it.